# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# cwatanab <chihiro.watanabe@live.jp>, 2016
# Ian Moone, 2014
# Francois REJETE <francois@rejete.com>, 2018
# INOUE Daisuke, 2025
# Masayuki Kumagai <makumagai@gmail.com>, 2016
msgid ""
msgstr ""
"Project-Id-Version: Modoboa\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-08-28 09:41+0200\n"
"PO-Revision-Date: 2011-07-01 08:15+0000\n"
"Last-Translator: INOUE Daisuke, 2025\n"
"Language-Team: Japanese (Japan) (http://app.transifex.com/tonio/modoboa/language/ja_JP/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ja_JP\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: admin/api/v1/serializers.py:39
msgid ""
"Quota shared between mailboxes. Can be expressed in KB, MB (default) or GB. "
"A value of 0 means no quota."
msgstr "すべてのメールボックス全体の容量制限。KB、MB（デフォルト）または GB で指定します。値を 0 にすると容量制限しないことを意味します。"

#: admin/api/v1/serializers.py:46
msgid "Default quota in MB applied to mailboxes. A value of 0 means no quota."
msgstr "メールボックスの容量制限のデフォルト値を MB で指定します。値を 0 にすると容量制限しません。"

#: admin/api/v1/serializers.py:94
msgid "domain alias with this name already exists"
msgstr "この名前のドメインエイリアスがすでに存在します"

#: admin/api/v1/serializers.py:104
msgid "No authorized MX record found for this domain"
msgstr "このドメインの承認済み MX レコードが見つかりません"

#: admin/api/v1/serializers.py:122
msgid "Cannot be greater than domain quota"
msgstr "ドメインの容量制限より大きくすることはできません"

#: admin/api/v1/serializers.py:168 admin/api/v1/serializers.py:364
#: admin/api/v2/serializers.py:564 admin/models/alias.py:44
msgid "Permission denied."
msgstr "権限がありません。"

#: admin/api/v1/serializers.py:231 admin/api/v2/serializers.py:90
#: admin/api/v2/serializers.py:486 core/api/v2/serializers.py:255
#: core/api/v2/serializers.py:257 core/api/v2/serializers.py:266
#: transport/backends.py:40
msgid "This field is required"
msgstr "このフィールドは必須です"

#: admin/api/v1/serializers.py:243
msgid ""
"List of administered domains (resellers and domain administrators only)."
msgstr "管理下ドメイン一覧 ( リセラーとドメイン管理者のみ )"

#: admin/api/v1/serializers.py:337 admin/api/v2/serializers.py:546
msgid "Not allowed for this role."
msgstr "このロールは許可されていません。"

#: admin/api/v1/serializers.py:355
msgid "Must be equal to mailbox full_address"
msgstr "メールボックスの full_address と同じにしてください"

#: admin/api/v1/serializers.py:397 admin/api/v2/serializers.py:598
#: calendars/serializers.py:175 calendars/serializers.py:177
#: pdfcredentials/api/v2/serializers.py:91
msgid "This field is required."
msgstr "このフィールドは必須です。"

#: admin/api/v1/serializers.py:409 admin/api/v2/serializers.py:615
msgid "Local domain {} does not exist"
msgstr "ローカルドメイン {}  は、存在しません"

#: admin/api/v1/serializers.py:499
msgid "A list of recipient"
msgstr "受信者一覧"

#: admin/api/v1/serializers.py:515 admin/api/v2/viewsets.py:286
msgid "This alias already exists"
msgstr "このエイリアスはすでに存在します"

#: admin/api/v1/serializers.py:551
msgid "You don't have access to this domain."
msgstr "このドメインにはアクセスできません。"

#: admin/api/v1/serializers.py:560
msgid "You don't have access to this mailbox."
msgstr "このメールボックスにアクセスできません。"

#: admin/api/v1/viewsets.py:184
msgid "Here is your new Modoboa password: {}"
msgstr "Modoboa の新しいパスワード : {}"

#: admin/api/v2/serializers.py:82
msgid "DKIM keys storage directory not configured"
msgstr "DKIM キー保存ディレクトリーを設定してください"

#: admin/api/v2/serializers.py:94
msgid "This field is valid when type is relaydomain"
msgstr "この項目はリレードメインの設定の際に有効です"

#: admin/api/v2/serializers.py:101
msgid "You can't define an unlimited quota."
msgstr "容量制限を無制限に設定できません。"

#: admin/api/v2/serializers.py:135
#, python-format
msgid "User '%s' already exists"
msgstr "ユーザー「%s」はすでに存在します"

#: admin/api/v2/serializers.py:250 admin/api/v2/serializers.py:256
msgid "Must be a positive integer"
msgstr "正の整数を入力してください"

#: admin/api/v2/serializers.py:265
msgid "openssl not found, please make sure it is installed."
msgstr "openssl が見つかりません。インストールを確認してください。"

#: admin/api/v2/serializers.py:268 pdfcredentials/api/v2/serializers.py:84
msgid "Directory not found."
msgstr "ディレクトリーが見つかりません。"

#: admin/api/v2/serializers.py:279
msgid "This field only allows valid IP addresses (or networks)"
msgstr "有効な IP アドレス ( またはネットワーク ) を入力してください"

#: admin/api/v2/serializers.py:292
msgid "Please disable imap migration before disabling this option."
msgstr "この設定を無効にする前に、IMAP 移行を無効にしてください。"

#: admin/api/v2/serializers.py:305
msgid "Define at least one authorized network / address"
msgstr "認証済みのネットワークかアドレスを少なくとも 1 つ以上定義してください"

#: admin/api/v2/serializers.py:581
msgid "The left part of an email address can't be more than 64 characters."
msgstr "メールアドレスの左部分を 64 文字以下にしてください。"

#: admin/api/v2/serializers.py:603
msgid "A mailbox is required to create aliases."
msgstr "エイリアスを作成するにはメールボックスが必要です。"

#: admin/api/v2/serializers.py:772
msgid "Unknown status"
msgstr "不明な状態"

#: admin/api/v2/viewsets.py:84
msgid "You don't have access to this domain"
msgstr "このドメインにはアクセスできません"

#: admin/api/v2/viewsets.py:87
msgid "You can't delete your own domain"
msgstr "自分自身のドメインは削除できません"

#: admin/app_settings.py:19 limits/app_settings.py:38 limits/constants.py:13
msgid "Domains"
msgstr "ドメイン"

#: admin/app_settings.py:25
msgid "Enable MX checks"
msgstr "MX チェック有効"

#: admin/app_settings.py:27
msgid "Check that every domain has a valid MX record"
msgstr "各ドメインに有効な MX レコードがあるかをチェックします"

#: admin/app_settings.py:34
msgid "Valid MXs"
msgstr "有効な MX"

#: admin/app_settings.py:37
msgid ""
"A list of IP or network address every MX record should match. A warning will"
" be sent if a record does not respect it."
msgstr "各 MX レコードにマッチすることを期待する IP またはネットワークアドレスのリスト。レコードが期待通りでない場合は、警告を送信します。"

#: admin/app_settings.py:47
msgid "New domains must use authorized MX(s)"
msgstr "新しいドメインには承認済みの MX 利用を強制"

#: admin/app_settings.py:51
msgid ""
"Prevent the creation of a new domain if its MX record does not use one of "
"the defined addresses."
msgstr "MX レコードが定義したアドレスのいずれかを使用していない場合、新しいドメインの作成を防止します。"

#: admin/app_settings.py:59
msgid "Enable IPV6 checks"
msgstr "IPV6 チェック有効"

#: admin/app_settings.py:62
msgid "Check if MX setup for ipv6 is valid"
msgstr "IPV6 用の MX の設定が有効かチェックします"

#: admin/app_settings.py:69
msgid "Enable SPF checks"
msgstr "SPF チェック有効"

#: admin/app_settings.py:71
msgid "Check if every domain has a valid SPF record"
msgstr "各ドメインに有効な SPF レコードがあるかをチェックします"

#: admin/app_settings.py:78
msgid "Enable DKIM checks"
msgstr "DKIM チェック有効"

#: admin/app_settings.py:80
msgid "Check if every domain with DKIM signin enabled has a valid DNS record"
msgstr "各 DKIM 署名のあるすべてのドメインに有効な DNS レコードがあるかをチェックします"

#: admin/app_settings.py:88
msgid "Enable DMARC checks"
msgstr "DMARC チェック有効"

#: admin/app_settings.py:90
msgid "Check if every domain has a valid DMARC record"
msgstr "各ドメインに有効な DMARC レコードがあるかをチェックします"

#: admin/app_settings.py:97
msgid "Enable autoconfig checks"
msgstr "自動構成チェック有効"

#: admin/app_settings.py:99
msgid "Check if every domain has a valid records for autoconfiguration"
msgstr "各ドメインに有効な自動構成のレコードがあるかをチェックします"

#: admin/app_settings.py:107
msgid "Enable DNSBL checks"
msgstr "DNSBL チェック有効"

#: admin/app_settings.py:109
msgid "Check every domain against major DNSBL providers"
msgstr "各ドメインを主な DNSBL でチェックします"

#: admin/app_settings.py:116
msgid "Custom DNS server"
msgstr "カスタム DNS サーバー"

#: admin/app_settings.py:118
msgid "Use a custom DNS server instead of local server configuration"
msgstr "ローカルサーバーで設定された DNS サーバーではなく、カスタム DNS サーバーを利用します"

#: admin/app_settings.py:126
msgid "DKIM keys storage directory"
msgstr "DKIM キー保存ディレクトリー"

#: admin/app_settings.py:128
msgid ""
"Absolute path of the directory where DKIM private keys will be stored. Make "
"sure this directory belongs to root user and is not readable by the outside "
"world."
msgstr "DKIM プライベートキーを保存するディレクトリーの絶対パス。このディレクトリーが root ユーザー所有で、外の世界からは読めないことを確認してください。"

#: admin/app_settings.py:137
msgid "Default DKIM key length"
msgstr "DKIM キー長のデフォルト"

#: admin/app_settings.py:139
msgid "Default length in bits for newly generated DKIM keys."
msgstr "新たに生成する DKIM キーの長さのデフォルト値です。"

#: admin/app_settings.py:146
msgid "Default domain quota"
msgstr "ドメイン容量制限のデフォルト"

#: admin/app_settings.py:148
msgid ""
"Default quota (in MB) applied to freshly created domains with no value "
"specified. A value of 0 means no quota."
msgstr "特に値をせずに新たにドメインを作成する際の容量制限のデフォルト値 ( MB ) 。値を 0 にすると容量制限しません。"

#: admin/app_settings.py:156
msgid "Default domain sending limit"
msgstr "ドメイン送信通数制限のデフォルト"

#: admin/app_settings.py:158
msgid ""
"Number of messages freshly created domains will be allowed to send per day. "
"Leave empty for no limit."
msgstr "新たにドメインを作成する際、1 日間に送信可能なメッセージ数。空にすると制限しません。"

#: admin/app_settings.py:170 limits/app_settings.py:61
#: limits/app_settings.py:135 limits/constants.py:31 limits/constants.py:84
msgid "Mailboxes"
msgstr "メールボックス"

#: admin/app_settings.py:176
msgid "Handle mailboxes on filesystem"
msgstr "ファイルシステムのメールボックスも操作"

#: admin/app_settings.py:178
msgid ""
"Rename or remove mailboxes on the filesystem when they get renamed or "
"removed within Modoboa"
msgstr "Modoboa でメールボックスを変名・削除すると、ファイルシステム上も変名・削除する"

#: admin/app_settings.py:186 admin/models/domain.py:36
msgid "Default mailbox quota"
msgstr "メールボックス容量制限のデフォルト"

#: admin/app_settings.py:188
msgid ""
"Default mailbox quota (in MB) applied to freshly created domains with no "
"value specified. A value of 0 means no quota."
msgstr "特に値をせずに新たにドメインを作成する際の、メールボックスの容量制限のデフォルト値 ( MB ) 。値を 0 にすると容量制限しません。"

#: admin/app_settings.py:197
msgid "Default mailbox sending limit"
msgstr "メールボックス送信通数制限のデフォルト"

#: admin/app_settings.py:199
msgid ""
"Number of messages freshly created mailboxes will be allowed to send per "
"day. Leave empty for no limit."
msgstr "新たにメールボックスを作成する際に設定する、1 日に送信できるメッセージ数。空にすると制限しません。"

#: admin/app_settings.py:207
msgid "Automatic account removal"
msgstr "自動アカウント削除"

#: admin/app_settings.py:209
msgid "When a mailbox is removed, also remove the associated account"
msgstr "メールボックスを削除すると、関連するアカウントも削除します"

#: admin/app_settings.py:218
msgid "Automatic domain/mailbox creation"
msgstr "自動ドメイン・メールボックス作成"

#: admin/app_settings.py:221
msgid ""
"Create a domain and a mailbox when an account is automatically created."
msgstr "アカウントを作成するとドメインとメールボックスも自動で作成します。"

#: admin/app_settings.py:230
msgid "Create an alias when a mailbox is renamed"
msgstr "メールボックスの変名でエイリアスを作成"

#: admin/app_settings.py:233
msgid "Create an alias using the old address when a mailbox is renamed."
msgstr "メールボックスの名前を変更する際、古いアドレスをエイリアスとして作成します。"

#: admin/app_settings.py:287 core/api/v2/viewsets.py:174
msgid "Administration"
msgstr "管理"

#: admin/constants.py:21
msgid "Domain"
msgstr "ドメイン"

#: admin/constants.py:22
msgid "Relay domain"
msgstr "リレードメイン"

#: admin/constants.py:38
msgid "Opened"
msgstr "要対応"

#: admin/constants.py:39
msgid "Closed"
msgstr "済"

#: admin/handlers.py:179
msgid "domain alias"
msgstr "ドメインエイリアス"

#: admin/handlers.py:227
#, python-format
msgid "Account import failed (%s): domain does not exist"
msgstr "アカウントインポート失敗 ( %s ) : ドメインがありません"

#: admin/handlers.py:242 admin/models/mailbox.py:342
msgid "Mailbox {} already exists"
msgstr "{} メールボックスはすでに存在します"

#: admin/handlers.py:251
#, python-format
msgid "Account import failed (%s): wrong quota value"
msgstr "アカウントインポート失敗 ( %s ) : 容量制限の値に誤り"

#: admin/lib.py:121
msgid "You are not allowed to import domains"
msgstr "ドメインをインポートする権限がありません"

#: admin/lib.py:130
msgid "You are not allowed to import domain aliases."
msgstr "ドメインエイリアスをインポートする権限がありません"

#: admin/lib.py:222 admin/tests/test_mx.py:153 admin/tests/test_mx.py:166
#: admin/tests/test_mx.py:171
#, python-format
msgid "No DNS record found for %s"
msgstr "%s の DNS レコードが見つかりません"

#: admin/lib.py:225
#, python-format
msgid "No %(type)s record for %(name)s"
msgstr "%(name)s の %(type)s レコードなし"

#: admin/lib.py:229 admin/tests/test_mx.py:156
msgid "No working name servers found"
msgstr "稼働しているサーバーが見つかりません"

#: admin/lib.py:232 admin/tests/test_mx.py:160
#, python-format
msgid "DNS resolution timeout, unable to query %s at the moment"
msgstr "DNS 名前解決でタイムアウトが発生し、今のところ %s の問い合わせはできません"

#: admin/lib.py:236
msgid "DNS name is too long: {}"
msgstr "DNS 名が長すぎます : {}"

#: admin/lib.py:267
#, python-brace-format
msgid "Invalid IP address format for {domain}; {addr}"
msgstr "{domain} の IP アドレス書式が無効です : {addr}"

#: admin/lib.py:360
#, python-format
msgid "Object already exists: %s"
msgstr "オブジェクトはすでに存在します : %s"

#: admin/lib.py:364
#, python-format
msgid "%d objects imported successfully"
msgstr "%d 個のオブジェクトをインポートしました"

#: admin/management/commands/subcommands/_import.py:97
msgid "CSV file is not encoded in UTF-8, attempting to guess encoding"
msgstr "CSV ファイルが UTF-8 でエンコードされていません。エンコードの推測を試みます"

#: admin/management/commands/subcommands/_import.py:112
#, python-format
msgid "Reading CSV file using %(encoding)s encoding"
msgstr "CSV ファイルを %(encoding)s エンコードで読んでいます"

#: admin/management/commands/subcommands/_import.py:123
#, python-format
msgid "Unable to decode CSV file using %(encoding)s encoding"
msgstr "%(encoding)s エンコードでは CSV ファイルをデコードできません"

#: admin/management/commands/subcommands/_manage_dkim_keys.py:29
msgid ""
"DKIM path non-writable (either a permission issue or the directory does not "
"exist)"
msgstr "DKIM パスが書き込みできません ( 権限問題か、ディレクトリーが存在しないかのいずれかです )"

#: admin/management/commands/subcommands/_manage_dkim_keys.py:52
msgid "Failed to generate DKIM private key"
msgstr "DKIM プライベートキーの生成に失敗"

#: admin/management/commands/subcommands/_manage_dkim_keys.py:62
msgid "Failed to generate DKIM public key"
msgstr "DKIM 公開キーの生成に失敗"

#: admin/management/commands/subcommands/_mx.py:133
msgid "MX {} listed by DNSBL provider {}"
msgstr "MX {} が DNSBL・{} に掲載されています"

#: admin/management/commands/subcommands/_mx.py:174
msgid "Domain has no MX record"
msgstr "ドメインに MX レコードがありません"

#: admin/management/commands/subcommands/_mx.py:177
msgid "Domain {} has no MX record"
msgstr "{} ドメインは MX レコードがありません"

#: admin/management/commands/subcommands/_mx.py:195
msgid "Invalid MX record: {}"
msgstr "無効な MX レコード : {}"

#: admin/management/commands/subcommands/_mx.py:202
#, python-brace-format
msgid "MX record for domain {0} is invalid: {1}"
msgstr "{0} ドメインの MX レコードが無効です : {1}"

#: admin/management/commands/subcommands/_mx.py:213
msgid "[modoboa] MX issue(s) for domain {}"
msgstr "[modoboa] {} ドメインの MX 問題"

#: admin/management/commands/subcommands/_mx.py:269
msgid "[modoboa] DNSBL issue(s) for domain {}"
msgstr "[modoboa] {} ドメインの DNSBL 問題"

#: admin/models/alias.py:42
msgid "Domain not found."
msgstr "ドメインが見つかりません。"

#: admin/models/alias.py:97 admin/models/mailbox.py:85
msgid "address"
msgstr "アドレス"

#: admin/models/alias.py:99
msgid "The alias address."
msgstr "エイリアスアドレス。"

#: admin/models/alias.py:103 admin/models/domain.py:45
#: admin/models/domain_alias.py:45 autoreply/models.py:25
msgid "enabled"
msgstr "有効"

#: admin/models/alias.py:104 admin/models/domain_alias.py:46
msgid "Check to activate this alias"
msgstr "このエイリアスを有効にする前にチェックする"

#: admin/models/alias.py:108
msgid "Description"
msgstr "説明"

#: admin/models/alias.py:109
msgid "Expire at"
msgstr "期限 : "

#: admin/models/alias.py:150
msgid "alias"
msgstr "エイリアス"

#: admin/models/alias.py:183
msgid "Invalid address"
msgstr "無効なアドレス"

#: admin/models/alias.py:202
msgid "Local recipient {}@{} not found"
msgstr "ローカル受信者 {}@{} は見つかりません"

#: admin/models/alias.py:251
msgid "Invalid line: {}"
msgstr "無効な行 : {}"

#: admin/models/domain.py:27 admin/models/domain_alias.py:33
msgid "name"
msgstr "名前"

#: admin/models/domain.py:30
msgid "The domain name"
msgstr "ドメイン名"

#: admin/models/domain.py:39 admin/models/mailbox.py:92
msgid "Message sending limit"
msgstr "メッセージ送信通数制限"

#: admin/models/domain.py:42
msgid "Number of messages this domain can send per day"
msgstr "このドメインから 1 日に送信できるメッセージ数"

#: admin/models/domain.py:46
msgid "Check to activate this domain"
msgstr "このドメインを有効にする前にチェックする"

#: admin/models/domain.py:51
msgid "Enable DNS checks"
msgstr "DNS チェック有効"

#: admin/models/domain.py:53
msgid "Check to enable DNS checks for this domain"
msgstr "このドメインを有効にする前に DNS チェックする"

#: admin/models/domain.py:61
msgid "Enable DKIM signing"
msgstr "DKIM 署名を有効"

#: admin/models/domain.py:63
msgid ""
"If you activate this feature, a DKIM key will be generated for this domain."
msgstr "この機能を有効にすると、このドメインの DKIM キーを生成します"

#: admin/models/domain.py:70
msgid "Key length"
msgstr "キー長"

#: admin/models/domain.py:321 admin/models/domain_alias.py:70
#: core/models.py:352 relaydomains/lib.py:13
msgid "Invalid line"
msgstr "行が無効"

#: admin/models/domain.py:331
msgid "{}: no authorized MX record found for domain"
msgstr "{} : ドメインの承認済み MX レコードが見つかりません"

#: admin/models/domain.py:337
msgid "{}: invalid quota value for domain"
msgstr "{} : ドメインの容量制限の値が無効です"

#: admin/models/domain.py:343
msgid "{}: invalid default mailbox quota value for domain"
msgstr "{} : ドメインの各メールボックスの容量制限の値が無効です"

#: admin/models/domain.py:350
msgid "{}: default mailbox quota cannot be greater than domain quota"
msgstr "{} : 各メールボックスの容量制限の値は、ドメインの容量制限の値より小さくしてください"

#: admin/models/domain_alias.py:36
msgid "The alias name"
msgstr "エイリアス名"

#: admin/models/domain_alias.py:40
msgid "target"
msgstr "対象"

#: admin/models/domain_alias.py:41
msgid "The domain this alias points to"
msgstr "このエイリアスの指定先ドメイン"

#: admin/models/domain_alias.py:79
#, python-format
msgid "Unknown domain %s"
msgstr "%s ドメインは不明です"

#: admin/models/mailbox.py:87
msgid "Mailbox address (without the @domain.tld part)"
msgstr "メールボックス アドレス ( @domain.tld のドメイン部分を除く )"

#: admin/models/mailbox.py:95
msgid "Number of messages this mailbox can send per day"
msgstr "このメールボックスから 1 日に送信できるメッセージ数"

#: admin/models/mailbox.py:153
#, python-format
msgid "Failed to retrieve mailbox location (%s)"
msgstr "メールボックスの場所 ( %s ) を取得できません"

#: admin/models/mailbox.py:249
msgid "A quota is required"
msgstr "容量制限の設定が必要です"

#: admin/models/mailbox.py:256
msgid "{}: domain quota exceeded"
msgstr "{} : ドメインの容量制限を超過"

#: admin/models/mailbox.py:310
msgid "Domain does not exist"
msgstr "ドメインがありません"

#: admin/templates/admin/notifications/domain_in_dnsbl.html:3
#, python-format
msgid ""
"Modoboa detected that domain %(domain)s is listed by the following DNSBL "
"providers:"
msgstr "Modoboa が %(domain)s ドメインの以下の DNSBL への掲載を検出 : "

#: admin/templates/admin/notifications/domain_in_dnsbl.html:11
msgid ""
"The domain's reputation will be affected and there is a chance that emails "
"coming from it are considered as spam. You should contact those providers "
"and ask them to unlist detected IP address(es)."
msgstr "ドメインの格付けが影響を与え、このドメインからのメールが SPAM とみなされる可能性があります。DNSBL 提供元に連絡し、検出された IP アドレスのリストからの削除を依頼する必要があります。"

#: admin/templates/admin/notifications/domain_invalid_mx.html:3
#, python-format
msgid "Modoboa detected that domain %(domain)s has no valid MX record:"
msgstr "Modoboa が %(domain)s ドメインの有効な MX レコードがないことを検出 : "

#: admin/tests/test_mx.py:155
#, python-format
msgid "No MX record for %s"
msgstr "%s の MX レコードなし"

#: admin/tests/test_mx.py:176 admin/tests/test_mx.py:182
#, python-format
msgid "Invalid IP address format for %(domain)s; %(addr)s"
msgstr "%(domain)s の IP アドレス書式が無効です ; %(addr)s"

#: admin/tests/test_mx.py:199
#, python-format
msgid "No AAAA record for %s"
msgstr "%s の AAAA レコードなし"

#: autoreply/api/v2/serializers.py:27
msgid "Value can't be less than 1 day"
msgstr "1 日以上の値を設定してください"

#: autoreply/api/v2/serializers.py:47
msgid "This date is over"
msgstr "過去の日付です"

#: autoreply/api/v2/serializers.py:50
msgid "Must be greater than start date"
msgstr "開始日より後を設定してください"

#: autoreply/app_settings.py:12 core/apps.py:21
#: imap_migration/app_settings.py:13 maillog/apps.py:14
#: pdfcredentials/app_settings.py:13 webmail/app_settings.py:11
msgid "General"
msgstr "全般"

#: autoreply/app_settings.py:18
msgid "Period between two auto-replies"
msgstr "自動返信の間隔"

#: autoreply/app_settings.py:20
msgid ""
"Number of days to wait before sending a new auto-reply to the same recipient"
msgstr "同じ受信者に対して新たな自動返信を送信するまでの待ち日数"

#: autoreply/app_settings.py:27 autoreply/app_settings.py:36
msgid "Default subject"
msgstr "件名のデフォルト"

#: autoreply/app_settings.py:29
msgid ""
"Default subject used when an auto-reply message is created automatically"
msgstr "自動応答メッセージを自動生成する際の件名のデフォルト"

#: autoreply/app_settings.py:38
#, python-format
msgid ""
"Default content used when an auto-reply message is created automatically. "
"The '%(name)s' macro will be replaced by the user's full name."
msgstr "自動応答メッセージを自動生成する際の本文のデフォルト値。「%(name)s」マクロはユーザーのフルネームに置き換えます。"

#: autoreply/apps.py:15
msgid "Automatic replies"
msgstr "自動応答"

#: autoreply/models.py:17
msgid "subject"
msgstr "件名"

#: autoreply/models.py:19
msgid "The subject that will appear in sent emails"
msgstr "送信メールの件名に記載します"

#: autoreply/models.py:22
msgid "content"
msgstr "本文"

#: autoreply/models.py:22
msgid "The content that will appear in sent emails"
msgstr "送信メールの本文に記載します"

#: autoreply/models.py:25
msgid "Activate/Deactivate your auto reply"
msgstr "自動応答の稼働・停止"

#: calendars/app_settings.py:11
msgid "Server settings"
msgstr "サーバー設定"

#: calendars/app_settings.py:17
msgid "Server URL"
msgstr "サーバー URL"

#: calendars/app_settings.py:19
msgid ""
"The URL of your Radicale server. It will be used to construct calendar URLs."
msgstr "Radicale サーバーのURL。カレンダーの URL の作成に使用します。"

#: calendars/app_settings.py:31
msgid "Rights management"
msgstr "権利情報管理"

#: calendars/app_settings.py:37
msgid "Rights file's path"
msgstr "権利情報のファイルパス"

#: calendars/app_settings.py:39
msgid "Path to the file that contains rights definition"
msgstr "権利情報定義を保存するファイルのパス"

#: calendars/app_settings.py:46
msgid "Allow calendars administration"
msgstr "カレンダー管理を許可"

#: calendars/app_settings.py:48
msgid "Allow domain administrators to manage user calendars (read and write)"
msgstr "ドメイン管理者にユーザーカレンダーの管理 ( 読み取り・書き込み ) を許可します"

#: calendars/app_settings.py:60 core/app_settings.py:676
#: rspamd/app_settings.py:45
msgid "Miscellaneous"
msgstr "その他"

#: calendars/app_settings.py:66
msgid "Maximum size of ICS files"
msgstr "ICS ファイルの最大サイズ"

#: calendars/app_settings.py:68
msgid ""
"Maximum size in bytes of imported ICS files (or KB, MB, GB if specified)"
msgstr "インポートする ICS ファイルの最大サイズをバイト ( または KB、MB、GB ) で指定"

#: calendars/app_settings.py:89 core/api/v2/viewsets.py:195
msgid "Calendars"
msgstr "カレンダー"

#: calendars/models.py:48 calendars/models.py:65 contacts/models.py:38
msgid "Server location is not set, please fix it."
msgstr "サーバーの場所を設定していないので登録してください。"

#: calendars/viewsets.py:197
msgid "Uploaded file is too big (max: {} bytes)"
msgstr "アップロード ファイルが大きすぎます ( 最大 : {} バイト )"

#: contacts/app_settings.py:11
msgid "Synchronization"
msgstr "同期"

#: contacts/app_settings.py:18
msgid "Synchonize address book using CardDAV"
msgstr "CardDAV でアドレス帳を同期します"

#: contacts/app_settings.py:21
msgid ""
"Choose to synchronize or not your address book using CardDAV. You will be "
"able to access your contacts from the outside."
msgstr "CardDAV を使ってアドレス帳の同期を行うか・行わないかを選んでください。同期すると外からアドレス帳にアクセスできます。"

#: contacts/app_settings.py:29
msgid "Synchronization frequency"
msgstr "同期する頻度"

#: contacts/app_settings.py:31
msgid "Interval in seconds between 2 synchronization requests"
msgstr "同期要求の間隔 ( 秒 )"

#: contacts/app_settings.py:51 core/api/v2/viewsets.py:185
msgid "Contacts"
msgstr "連絡先"

#: contacts/constants.py:6 contacts/constants.py:12
msgid "Home"
msgstr "自宅"

#: contacts/constants.py:7 contacts/constants.py:13
msgid "Work"
msgstr "職場"

#: contacts/constants.py:8 contacts/constants.py:14
msgid "Other"
msgstr "その他"

#: contacts/constants.py:15
msgid "Main"
msgstr "メイン"

#: contacts/constants.py:16
msgid "Cellular"
msgstr "携帯電話"

#: contacts/constants.py:17
msgid "Fax"
msgstr "FAX"

#: contacts/constants.py:18
msgid "Pager"
msgstr "ポケベル"

#: contacts/serializers.py:101
msgid "Name or display name required"
msgstr "名前か表示名は必須です"

#: contacts/serializers.py:233
msgid "Minimum allowed value is 60s"
msgstr "60 秒以上を設定してください"

#: core/api/v1/serializers.py:68
msgid "Invalid PIN code"
msgstr "PIN コードが無効です"

#: core/api/v1/serializers.py:74
msgid "2FA is already enabled"
msgstr "2FA はすでに有効です"

#: core/api/v1/serializers.py:86
#, python-format
msgid "Failed TFA settings editing attempt from '%(addr)s' as user '%(user)s'"
msgstr "「%(addr)s」から「%(user)s」ユーザーの 2FA 設定変更は失敗"

#: core/api/v1/serializers.py:91 core/api/v2/serializers.py:445
msgid "Invalid password"
msgstr "無効なパスワード"

#: core/api/v1/viewsets.py:45 core/api/v1/viewsets.py:64
msgid "2FA is not enabled"
msgstr "2 要素認証が無効"

#: core/api/v2/serializers.py:69
msgid "Local"
msgstr "ローカル"

#: core/api/v2/serializers.py:194 core/app_settings.py:17
msgid "User profile"
msgstr "ユーザープロフィール"

#: core/api/v2/serializers.py:213 core/api/v2/serializers.py:221
#: core/api/v2/serializers.py:228
msgid "Invalid syntax"
msgstr "不正な構文"

#: core/api/v2/serializers.py:233
msgid "Invalid rounds number"
msgstr "不正なラウンド数"

#: core/api/v2/serializers.py:433 core/forms.py:81
msgid "This code is invalid"
msgstr "このコードは無効です"

#: core/api/v2/serializers.py:503 core/views/auth.py:194
#: core/views/auth.py:247
msgid "Please use the following code to recover your Modoboa password: {}"
msgstr "Modoboa のパスワード再設定で利用するコード : {}"

#: core/api/v2/viewsets.py:176
msgid "Administration console"
msgstr "管理コンソール"

#: core/api/v2/viewsets.py:187
msgid "Address book"
msgstr "アドレス帳"

#: core/api/v2/viewsets.py:197
msgid "Calendar"
msgstr "カレンダー"

#: core/api/v2/viewsets.py:205 core/api/v2/viewsets.py:207
#: webmail/app_settings.py:258 webmail/app_settings.py:265
msgid "Webmail"
msgstr "Web メール"

#: core/app_settings.py:37
msgid "Authentication"
msgstr "認証"

#: core/app_settings.py:43
msgid "Authentication type"
msgstr "認証の種類"

#: core/app_settings.py:45
msgid "The backend used for authentication"
msgstr "認証バックエンド"

#: core/app_settings.py:52
msgid "Default password scheme"
msgstr "デフォルトのパスワードスキーマ"

#: core/app_settings.py:54
msgid "Scheme used to crypt mailbox passwords"
msgstr "メールボックス パスワードを暗号化する際のスキーマ"

#: core/app_settings.py:61
msgid "Rounds"
msgstr "ラウンド"

#: core/app_settings.py:63
msgid ""
"Number of rounds to use (only used by sha256crypt and sha512crypt). Must be "
"between 1000 and 999999999, inclusive."
msgstr "ラウンド数 ( sha256crypt と sha512crypt のみ使用 ) 。1000 ～ 999999999 の間で設定してください。"

#: core/app_settings.py:73
msgid "Update password scheme at login"
msgstr "ログイン時にパスワード スキーマを更新"

#: core/app_settings.py:76
msgid "Update user password at login to use the default password scheme"
msgstr "デフォルトのパスワード スキーマに切り替えるため、ログイン時にパスワードを更新します"

#: core/app_settings.py:84
msgid "Default password"
msgstr "デフォルト パスワード"

#: core/app_settings.py:86
msgid "Default password for automatically created accounts."
msgstr "アカウント自動作成で使用するデフォルト パスワード"

#: core/app_settings.py:93
msgid "Random password length"
msgstr "ランダムパスワード長"

#: core/app_settings.py:95
msgid "Length of randomly generated passwords."
msgstr "ランダムで生成するパスワードの長さ。"

#: core/app_settings.py:103
msgid "Allow special characters for random password"
msgstr "ランダムパスワードに特殊文字を許可"

#: core/app_settings.py:106
msgid "Enable special characters in randomly generated passwords."
msgstr "ランダムに生成するパスワードに特殊文字を含めます"

#: core/app_settings.py:114
msgid "Update password service URL"
msgstr "パスワードサービス URL を更新"

#: core/app_settings.py:116
msgid ""
"The URL of an external page where users will be able to update their "
"password. It applies only to non local users, ie. those automatically "
"created after a successful external authentication (LDAP, SMTP)."
msgstr "ユーザがパスワードを更新するための外部ページの URL。ローカルユーザー以外のユーザー、つまり外部認証 ( LDAP、SMTP ) が成功した後に自動作成されるユーザにのみ適用されます。"

#: core/app_settings.py:126
msgid "Password recovery announcement"
msgstr "パスワード再設定の案内"

#: core/app_settings.py:128
msgid "A temporary message that will be displayed on the reset password page."
msgstr "パスワード再設定ページに表示する一時的なメッセージ"

#: core/app_settings.py:137
msgid "Enable password recovery by SMS"
msgstr "SMS を使ったパスワード再設定"

#: core/app_settings.py:140
msgid "Enable password recovery by SMS for users who filled a phone number."
msgstr "電話番号を入力したユーザーは SMS によるパスワード再設定を有効にする"

#: core/app_settings.py:148
msgid "SMS provider"
msgstr "SMS 事業者"

#: core/app_settings.py:151
msgid "Choose a provider to send password recovery SMS"
msgstr "パスワード再設定の SMS を送信する業者を選んでください"

#: core/app_settings.py:163
msgid "LDAP"
msgstr "LDAP"

#: core/app_settings.py:169 sievefilters/app_settings.py:18
#: sievefilters/app_settings.py:51 webmail/app_settings.py:36
#: webmail/app_settings.py:73
msgid "Server address"
msgstr "サーバーアドレス"

#: core/app_settings.py:171
msgid "The IP address or the DNS name of the LDAP server"
msgstr "LDAP サーバーの IP アドレスまたは DNS 名"

#: core/app_settings.py:178 sievefilters/app_settings.py:25
#: sievefilters/app_settings.py:67 webmail/app_settings.py:54
#: webmail/app_settings.py:91
msgid "Server port"
msgstr "サーバーポート"

#: core/app_settings.py:180
msgid "The TCP port number used by the LDAP server"
msgstr "LDAP サーバーの TCP ポート番号"

#: core/app_settings.py:188
msgid "Enable secondary server (fallback)"
msgstr "セカンダリーサーバー ( フォールバック ) を有効にする"

#: core/app_settings.py:191
msgid ""
"Enable a secondary LDAP server which will be used if the primary one fails"
msgstr "セカンダリー LDAP サーバー。プライマリーサーバーで失敗した際に利用します。"

#: core/app_settings.py:199
msgid "Secondary server address"
msgstr "セカンダリーサーバー アドレス"

#: core/app_settings.py:202
msgid "The IP address or the DNS name of the seondary LDAP server"
msgstr "セカンダリー LDAP サーバーの IP アドレスまたは DNS 名"

#: core/app_settings.py:209
msgid "Secondary server port"
msgstr "セカンダリーサーバー ポート"

#: core/app_settings.py:212
msgid "The TCP port number used by the LDAP secondary server"
msgstr "セカンダリー LDAP サーバーの TCP ポート番号"

#: core/app_settings.py:219 sievefilters/app_settings.py:58
#: webmail/app_settings.py:45
msgid "Use a secured connection"
msgstr "暗号化接続を使う"

#: core/app_settings.py:221
msgid "Use an SSL/STARTTLS connection to access the LDAP server"
msgstr "LDAP サーバー に接続する際に SSL/STARTTLS を使います"

#: core/app_settings.py:228
msgid "Active Directory"
msgstr "Active Directory"

#: core/app_settings.py:230
msgid "Tell if the LDAP server is an Active Directory one"
msgstr "LDAP サーバーは Active Directory ですか？"

#: core/app_settings.py:237
msgid "Administrator groups"
msgstr "管理者グループ"

#: core/app_settings.py:239
msgid ""
"Members of those LDAP Posix groups will be created as domain administrators."
" Use ';' characters to separate groups."
msgstr "ここで指定する LDAP の Posix グループのメンバーをドメイン管理者として作成します。複数のグループを指定する際は「;」で区切ってください。"

#: core/app_settings.py:248
msgid "Group type"
msgstr "グループタイプ"

#: core/app_settings.py:250
msgid "The LDAP group type to use with your directory."
msgstr "ディレクトリーで使用している LDAP グループのタイプ。"

#: core/app_settings.py:257
msgid "Groups search base"
msgstr "グループの検索ベース"

#: core/app_settings.py:259
msgid "The distinguished name of the search base used to find groups"
msgstr "グループを探す際の検索ベースの識別名 ( DN )"

#: core/app_settings.py:267
msgid "Password attribute"
msgstr "パスワード属性"

#: core/app_settings.py:269
msgid "The attribute used to store user passwords"
msgstr "ユーザーパスワードを格納している属性"

#: core/app_settings.py:276
msgid "LDAP authentication settings"
msgstr "LDAP 認証設定"

#: core/app_settings.py:284
msgid "Authentication method"
msgstr "認証方式"

#: core/app_settings.py:287
msgid "Choose the authentication method to use"
msgstr "使用する認証方式を選択"

#: core/app_settings.py:294 core/app_settings.py:357
msgid "Bind DN"
msgstr "バインド DN"

#: core/app_settings.py:296 core/app_settings.py:359
msgid ""
"The distinguished name to use when binding to the LDAP server. Leave empty "
"for an anonymous bind"
msgstr "LDAP サーバーにバインドする際の識別名。空のままにすると匿名バインドです。"

#: core/app_settings.py:305 core/app_settings.py:367
msgid "Bind password"
msgstr "バインド パスワード"

#: core/app_settings.py:307 core/app_settings.py:369
msgid "The password to use when binding to the LDAP server (with 'Bind DN')"
msgstr "(「バインド DN」で ) LDAP サーバーにバインドする際のパスワード"

#: core/app_settings.py:317 core/app_settings.py:422
msgid "Users search base"
msgstr "ユーザーの検索ベース"

#: core/app_settings.py:319 core/app_settings.py:424
msgid "The distinguished name of the search base used to find users"
msgstr "ユーザーを探す際の検索ベースの識別名 ( DN )"

#: core/app_settings.py:328 core/app_settings.py:433
msgid "Search filter"
msgstr "検索フィルター"

#: core/app_settings.py:330 core/app_settings.py:435
msgid ""
"An optional filter string (e.g. '(objectClass=person)'). In order to be "
"valid, it must be enclosed in parentheses."
msgstr "オプションのフィルタ文字列 ( 例 : (objectClass=person) )。「()」カッコで囲まないと有効になりません。"

#: core/app_settings.py:339
msgid "User DN template"
msgstr "ユーザー DN テンプレート"

#: core/app_settings.py:341
#, python-format
msgid ""
"The template used to construct a user's DN. It should contain one "
"placeholder (ie. %(user)s)"
msgstr "ユーザー DN を作成する際のテンプレート。1 つのプレースホルダー ( 例 : %(user)s ) を含めてください。"

#: core/app_settings.py:350
msgid "LDAP synchronization settings"
msgstr "LDAP 同期設定"

#: core/app_settings.py:378
msgid "Enable export to LDAP"
msgstr "LDAP へのエクスポートを有効"

#: core/app_settings.py:380
msgid ""
"Enable automatic synchronization between local database and LDAP directory"
msgstr "ローカル データベース・LDAP ディレクトリー間での自動同期を有効にする"

#: core/app_settings.py:389
msgid "Delete remote LDAP account when local account is deleted"
msgstr "ローカルアカウントを削除するとリモート LDAP のアカウントも削除する"

#: core/app_settings.py:392
msgid ""
"Delete remote LDAP account when local account is deleted, otherwise it will "
"be disabled."
msgstr "ローカルアカウントを削除すると、リモートLDAPアカウントも削除、または無効にします。"

#: core/app_settings.py:401
msgid "Account DN template"
msgstr "アカウント DN テンプレート"

#: core/app_settings.py:403
#, python-format
msgid ""
"The template used to construct an account's DN. It should contain one "
"placeholder (ie. %(user)s)"
msgstr "アカウント DN を作成する際のテンプレート。1 つのプレースホルダー ( 例 : %(user)s ) を含めてください。"

#: core/app_settings.py:412
msgid "Enable import from LDAP"
msgstr "LDAP からのインポートを有効"

#: core/app_settings.py:414
msgid "Enable account synchronization from LDAP directory to local database"
msgstr "LDAP ディレクトリーからローカル データベースへのアカウント同期を有効にします"

#: core/app_settings.py:444
msgid "Username attribute"
msgstr "ユーザー名属性"

#: core/app_settings.py:446
msgid "The name of the LDAP attribute where the username can be found."
msgstr "ユーザー名を格納している LDAP 属性名。"

#: core/app_settings.py:455
msgid "Enable Dovecot LDAP sync"
msgstr "Dovecot LDAP 同期を有効"

#: core/app_settings.py:457
msgid "LDAP authentication settings will be applied to Dovecot configuration."
msgstr "LDAP 認証設定を Dovecot の設定に適用します。"

#: core/app_settings.py:465
msgid "Dovecot LDAP config file"
msgstr "Dovecot LDAP 設定ファイル"

#: core/app_settings.py:467
msgid ""
"Location of the configuration file which contains Dovecot LDAP settings."
msgstr "Dovecot の LDAP 設定ファイルの場所。"

#: core/app_settings.py:480
msgid "Dashboard"
msgstr "ダッシュボード"

#: core/app_settings.py:486
msgid "Custom welcome message"
msgstr "ようこそメッセージのカスタマイズ"

#: core/app_settings.py:488
msgid "A welcome message that will be displayed instead the default one"
msgstr "ようこそメッセージをデフォルトのものから変更します"

#: core/app_settings.py:495
msgid "Custom RSS feed"
msgstr "カスタム RSS フィード"

#: core/app_settings.py:497
msgid "Display custom RSS feed to resellers and domain administrators"
msgstr "リセラーとドメイン管理者に表示する独自 RSS フィードを設定します。"

#: core/app_settings.py:506
msgid "Display custom RSS feed to super admins"
msgstr "スーパー管理者にカスタマイズした RSS フィードを表示します"

#: core/app_settings.py:509
msgid "Display custom RSS feed also to super administrators"
msgstr "カスタマイズした RSS フィードをスーパー管理者にも表示します"

#: core/app_settings.py:517
msgid "Hide features widget"
msgstr "機能ウィジェットを非表示に"

#: core/app_settings.py:519
msgid "Hide features widget for resellers and domain administrators"
msgstr "リセラーとドメイン管理者の機能ウィジェットを非表示にします"

#: core/app_settings.py:531
msgid "Theme"
msgstr "テーマ"

#: core/app_settings.py:537
msgid "Primary theme color"
msgstr "主要なテーマカラー"

#: core/app_settings.py:539
msgid ""
"Primary color of the theme (hexadecimal code), used as the background color "
"of the left menu for example"
msgstr "テーマの主要な色 ( 16進数コードで指定 )で、左メニューの背景色などに使います。"

#: core/app_settings.py:548
msgid "Primary theme color (lighter version)"
msgstr "主要なテーマカラー ( ライトバージョン )"

#: core/app_settings.py:551 core/app_settings.py:563
msgid "Primary color of the theme (hexadecimal code)"
msgstr "主要なテーマカラー ( 16進数カード )"

#: core/app_settings.py:560
msgid "Primary theme color (darker version)"
msgstr "主要なテーマカラー ( ダークバージョン )"

#: core/app_settings.py:571
msgid "Secondary theme color"
msgstr "サブのテーマカラー"

#: core/app_settings.py:573
msgid ""
"Secondary color of the theme (hexadecimal code), used as the background "
"color of some buttons for example"
msgstr "テーマのサブの色 ( 16進数コードで指定 ) で、いくつかのボタンの背景色などに使います。"

#: core/app_settings.py:581
msgid "Labels theme color"
msgstr "ラベルテーマカラー"

#: core/app_settings.py:583
msgid "Theme color for labels (hexadecimal code)"
msgstr "ラベルのテーマカラー ( 16進数コード )"

#: core/app_settings.py:595
msgid "Notifications"
msgstr "通知"

#: core/app_settings.py:601
msgid "Sender address"
msgstr "送信者アドレス"

#: core/app_settings.py:603
msgid "Email address used to send notifications."
msgstr "通知を送信するメールアドレスです。"

#: core/app_settings.py:614
msgid "Public API"
msgstr "パブリック API"

#: core/app_settings.py:620
msgid "Enable communication"
msgstr "通信を有効に"

#: core/app_settings.py:622 core/app_settings.py:632
msgid "Automatically checks if a newer version is available"
msgstr "新しいバージョンのリリースを自動チェックする"

#: core/app_settings.py:629
msgid "Check new versions"
msgstr "新しいバージョンをチェックする"

#: core/app_settings.py:640
msgid "Send an email when new versions are found"
msgstr "新バージョンリリースでメールを送信"

#: core/app_settings.py:644
msgid "Send an email to notify admins about new versions"
msgstr "新しいバージョンがリリースされたら管理者にメールで通知"

#: core/app_settings.py:651 sievefilters/constants.py:25
msgid "Recipient"
msgstr "受信者"

#: core/app_settings.py:654
msgid "Recipient of new versions notification emails."
msgstr "新しいバージョンのリリースを通知するメールの受信者を指定してください。"

#: core/app_settings.py:661
msgid "Send statistics"
msgstr "統計を送信"

#: core/app_settings.py:664
msgid "Send statistics to Modoboa public API (counters and used extensions)"
msgstr "Modoboa パブリック API の統計 ( 数と利用拡張機能 ) を送信します"

#: core/app_settings.py:683
msgid "Enable inactive account tracking"
msgstr "非利用アカウント追跡を有効に"

#: core/app_settings.py:686
msgid ""
"Allow the administrator to set a threshold (in days) beyond which an account"
" is considered inactive if the user hasn't logged in"
msgstr "ユーザーがログインしておらず、アカウント非利用とみなす期間 ( 日 ) を設定することを管理者に許可します"

#: core/app_settings.py:695
msgid "Inactive account threshold"
msgstr "アカウントを非利用とみなすログイン間隔"

#: core/app_settings.py:698
msgid ""
"An account with a last login date greater than this threshold (in days) will"
" be considered as inactive"
msgstr "最終ログイン日が、このしきい値 ( 日数 ) よりも以前になったアカウントを非利用とみなします"

#: core/app_settings.py:707
msgid "Top notifications check interval"
msgstr "ページトップ通知のチェック間隔"

#: core/app_settings.py:710
msgid "Interval between two top notification checks (in seconds)"
msgstr "2 つのページトップの通知のチェックの間隔 ( 秒 )"

#: core/app_settings.py:717
msgid "Maximum log record age"
msgstr "最大ログ記録期間"

#: core/app_settings.py:719
msgid "The maximum age in days of a log record"
msgstr "ログを記録する日数の最大期間"

#: core/app_settings.py:727
msgid "Retention time in message history"
msgstr "メッセージ履歴の保持期間"

#: core/app_settings.py:730
msgid "Retention time (in days) of a message in the message history section"
msgstr "メッセージ履歴セクションにメッセージを保持する期間 ( 単位 : 日 )"

#: core/app_settings.py:737
msgid "Items per page"
msgstr "ページあたりの件数"

#: core/app_settings.py:739
msgid "Number of displayed items per page"
msgstr "ページあたりに表示される項目の数"

#: core/app_settings.py:746
msgid "Default top redirection"
msgstr "デフォルトのリダイレクト先"

#: core/app_settings.py:748
msgid "The default redirection used when no application is specified"
msgstr "アプリケーションが未指定の際、デフォルトとなるリダイレクト先"

#: core/checks.py:15
msgid ""
"You have USE_TZ set to False, this may result in issues during transitions "
"between summer/winter time (ie the same local time occuring twice due to "
"clock change)."
msgstr "USE_TZ を False にすると、夏時間・冬時間の移行の際に問題となる可能性があります ( 例 : 時計が変更されることで、同じローカル時刻が 2 度発生します )。"

#: core/checks.py:19
msgid "Set `USE_TZ = True` in settings.py"
msgstr "「USE_TZ=True」を settings.py に設定してください"

#: core/checks.py:23
msgid "The password scheme you are using is deprecated."
msgstr "ご利用のパスワード スキーマは現在非推奨です。"

#: core/checks.py:24
msgid "Upgrade your password scheme to a stronger one"
msgstr "パスワード スキーマをより強力なものに更新してください"

#: core/checks.py:30
msgid ""
"Oauth2 application for dovecot has not been found. Webmail and some other "
"features won't be available."
msgstr "dovect 用の OAuth2 アプリが見つかりません。Web メールなど、いくつかの機能が使えません。"

#: core/checks.py:34
msgid ""
"Visit : "
"https://modoboa.readthedocs.io/en/latest/manual_installation/dovecot.html#dovecot-"
"oauth2"
msgstr "以下、参照 : https://modoboa.readthedocs.io/en/latest/manual_installation/dovecot.html#dovecot-oauth2"

#: core/checks.py:41
msgid ""
"Oauth2 application for radicale has not been found. Calendar and some other "
"features won't be available."
msgstr "radicale 用の OAuth2 アプリが見つかりません。カレンダーなど、いくつかの機能が使えません。"

#: core/checks.py:45
msgid ""
"Visit :  "
"https://modoboa.readthedocs.io/en/latest/manual_installation/radicale.html#radicale-"
"oauth2"
msgstr "以下、参照 : https://modoboa.readthedocs.io/en/latest/manual_installation/radicale.html#radicale-oauth2"

#: core/constants.py:6
msgid "Simple user"
msgstr "標準ユーザー"

#: core/constants.py:7
msgid "Domain administrator"
msgstr "ドメイン管理者"

#: core/constants.py:8
msgid "Reseller"
msgstr "リセラー"

#: core/constants.py:9
msgid "Super administrator"
msgstr "スーパー管理者"

#: core/constants.py:53
msgid "No"
msgstr "いいえ"

#: core/constants.py:59
msgid "Search and bind"
msgstr "検索とバインド"

#: core/constants.py:60
msgid "Direct bind"
msgstr "ダイレクトバインド"

#: core/constants.py:109
msgid "Choose a provider"
msgstr "事業者を選択"

#: core/constants.py:114
msgid "Dummy"
msgstr "ダミー"

#: core/forms.py:56
msgid "Verification code"
msgstr "確認コード"

#: core/forms.py:67
msgid "Invalid code"
msgstr "無効なコード"

#: core/handlers.py:36
msgid "added"
msgstr "追加しました"

#: core/handlers.py:39
msgid "modified"
msgstr "変更しました"

#: core/handlers.py:41
#, python-format
msgid "%(object)s '%(name)s' %(action)s by user %(user)s"
msgstr "%(object)s '%(name)s' %(action)s by %(user)s ユーザー"

#: core/handlers.py:64
#, python-format
msgid "%(object)s '%(name)s' %(action)s by "
msgstr "%(object)s '%(name)s' %(action)s by "

#: core/handlers.py:67
msgid "deleted"
msgstr "削除済み"

#: core/handlers.py:71
msgid "user {}"
msgstr "{} ユーザー"

#: core/handlers.py:73
msgid "management command"
msgstr "管理コマンド"

#: core/handlers.py:95
msgid "You can't delete your own account"
msgstr "自分自身のアカウントは削除できません"

#: core/handlers.py:127
msgid "One or more updates are available"
msgstr "アップデートが利用できます"

#: core/handlers.py:138
#, python-format
msgid "You are still using a deprecated password scheme (%s)"
msgstr "非推奨のパスワード スキーマ ( %s ) を利用しています"

#: core/management/commands/communicate_with_public_api.py:42
msgid "[modoboa] Update(s) available"
msgstr "[modoboa] 新バージョン リリース"

#: core/models.py:71
msgid "Allow mailboxes access"
msgstr "メールボックスのアクセスを許可"

#: core/models.py:73
msgid "Allow this administrator to access user mailboxes"
msgstr "この管理者にユーザーのメールボックスへのアクセスを許可します"

#: core/models.py:75
msgid "password"
msgstr "パスワード"

#: core/models.py:78
msgid "language"
msgstr "言語"

#: core/models.py:82
msgid "Prefered language to display pages."
msgstr "ページ表示で優先する言語。"

#: core/models.py:84
msgid "Phone number"
msgstr "電話番号"

#: core/models.py:86
msgid "Secondary email"
msgstr "別のメールアドレス"

#: core/models.py:91
msgid "An alternative e-mail address, can be used for recovery needs."
msgstr "別のメールアドレス。再設定が必要な際に利用します。"

#: core/models.py:150
msgid "Failed to update password: LDAP module not installed"
msgstr "パスワードの更新に失敗 : LDAP モジュールがインストールされていません"

#: core/models.py:185 policyd/core.py:117
msgid "account"
msgstr "アカウント"

#: core/models.py:317
msgid "Unknown"
msgstr "不明"

#: core/models.py:360
msgid "You can't import an account with a role greater than yours"
msgstr "自分のロール以上のアカウントはインポートできません"

#: core/models.py:375
msgid "The simple user '{}' must have a valid email address"
msgstr "標準ユーザー「{}」に有効なメールアドレスがありません"

#: core/models.py:383
msgid "username and email fields must not differ for '{}'"
msgstr "「{}」では、ユーザー名とメールアドレスの項目は同一にしてください。"

#: core/password_validation.py:25
msgid "Password must contain at least {} digit."
msgid_plural "Password must contain at least {} digits."
msgstr[0] "パスワードに {} 文字以上の数字を含めてください。"

#: core/password_validation.py:37
msgid "Password must contain at least {} lowercase letter."
msgid_plural "Password must contain at least {} lowercase letters."
msgstr[0] "パスワードに {} 文字以上のアルファベット小文字を含めてください。"

#: core/password_validation.py:49
msgid "Password must contain at least {} uppercase letter."
msgid_plural "Password must contain at least {} uppercase letters."
msgstr[0] "パスワードに {} 文字以上のアルファベット大文字を含めてください。"

#: core/password_validation.py:62
msgid "Password must contain at least {} special character."
msgid_plural "Password must contain at least {} special characters."
msgstr[0] "パスワードに {} 文字以上の記号を含めてください。"

#: core/password_validation.py:70
msgid "Your password must contain a combination of different character types."
msgstr "パスワードはさまざまな種類の文字を組み合わせる必要があります。"

#: core/sms_backends/ovh.py:21 core/sms_backends/ovh.py:102
msgid "API endpoint"
msgstr "API エンドポイント"

#: core/sms_backends/ovh.py:24 core/sms_backends/ovh.py:62
msgid "OVH Europe"
msgstr "OVH Europe"

#: core/sms_backends/ovh.py:25 core/sms_backends/ovh.py:63
msgid "OVH US"
msgstr "OVH US"

#: core/sms_backends/ovh.py:26 core/sms_backends/ovh.py:64
msgid "OVH North-America"
msgstr "OVH North-America"

#: core/sms_backends/ovh.py:27 core/sms_backends/ovh.py:65
msgid "So you Start Europe"
msgstr "So you Start Europe"

#: core/sms_backends/ovh.py:28 core/sms_backends/ovh.py:66
msgid "So you Start North America"
msgstr "So you Start North America"

#: core/sms_backends/ovh.py:29 core/sms_backends/ovh.py:67
msgid "Kimsufi Europe"
msgstr "Kimsufi Europe"

#: core/sms_backends/ovh.py:30 core/sms_backends/ovh.py:68
msgid "Kimsufi North America"
msgstr "Kimsufi North America"

#: core/sms_backends/ovh.py:36 core/sms_backends/ovh.py:109
msgid "Application key"
msgstr "アプリケーションキー"

#: core/sms_backends/ovh.py:41
msgid "Application secret"
msgstr "アプリケーションシークレット"

#: core/sms_backends/ovh.py:49 core/sms_backends/ovh.py:124
msgid "Consumer key"
msgstr "カスタマーキー"

#: core/sms_backends/ovh.py:116
msgid "Appication secret"
msgstr "アプリケーションシークレット"

#: core/templates/core/notifications/update_available.html:3
msgid ""
"\n"
"  The following update is available:\n"
msgid_plural ""
"\n"
"  The following updates are available:\n"
msgstr[0] "\n  以下の更新が用意されています :\n"

#: core/utils.py:43
msgid "The core part of Modoboa"
msgstr "Modoboa のコア部分"

#: core/views/auth.py:65
#, python-format
msgid "User '%s' successfully logged in"
msgstr "ユーザー「%s」がログインしました"

#: core/views/auth.py:109
#, python-format
msgid "Password scheme mismatch. Updating %s password"
msgstr "パスワード スキーマが一致しません。%s のパスワードを更新しています"

#: core/views/auth.py:116
#, python-format
msgid "Password hash parameter missmatch. Updating %s password"
msgstr "パスワードのハッシュ パラメーターが一致しません。%s パスワードを更新しています"

#: core/views/auth.py:154
msgid "User '{}' successfully logged out"
msgstr "ユーザー「{}」がログアウトしました"

#: dmarc/forms.py:15
msgid "DNS settings"
msgstr "DNS 設定"

#: dmarc/forms.py:21
msgid "Enable reverse lookups"
msgstr "逆引きを有効に"

#: dmarc/forms.py:23
msgid "Enable reverse DNS lookups (reports will be longer to display)"
msgstr "DNS の逆引きを有効にします ( レポート表示まで時間がかかるようになります )"

#: dmarc/forms.py:42
msgid "DMARC"
msgstr "DMARC"

#: dmarc/lib.py:294
msgid "Not resolved"
msgstr "未解決"

#: dmarc/models.py:10 dmarc/models.py:16 pdfcredentials/constants.py:7
#: webmail/constants.py:18
msgid "None"
msgstr "なし"

#: dmarc/models.py:11
msgid "Quarantine"
msgstr "検疫"

#: dmarc/models.py:12
msgid "Reject"
msgstr "拒否"

#: dmarc/models.py:17
msgid "Neutral"
msgstr "中立"

#: dmarc/models.py:18
msgid "Pass"
msgstr "パス"

#: dmarc/models.py:19
msgid "Fail"
msgstr "失敗"

#: dmarc/models.py:20
msgid "Temporary error"
msgstr "一時的なエラー"

#: dmarc/models.py:21
msgid "Permanent error"
msgstr "永続的なエラー"

#: dmarc/models.py:24
msgid "Policy"
msgstr "ポリシー"

#: dmarc/models.py:26
msgid "Soft failure"
msgstr "ソフトフェイル"

#: dnstools/lib.py:82
msgid "Wrong ip4 mechanism syntax"
msgstr "IP4 メカニズム書式に誤り"

#: dnstools/lib.py:86
msgid "Wrong IPv4 address format"
msgstr "IP4 アドレス書式に誤り"

#: dnstools/lib.py:92
msgid "Wrong ip6 mechanism syntax"
msgstr "IPV6 メカニズム書式に誤り"

#: dnstools/lib.py:97
msgid "Wrong IPv6 address format"
msgstr "IPV6 アドレス書式に誤り"

#: dnstools/lib.py:111 dnstools/lib.py:136
msgid "Invalid syntax for {} mechanism"
msgstr "{} メカニズムの書式が無効"

#: dnstools/lib.py:113
msgid "Invalid mask found {}"
msgstr "{} に無効なマスク"

#: dnstools/lib.py:157
msgid "Not an SPF record"
msgstr "SPF レコードではありません"

#: dnstools/lib.py:177
msgid "Unknown mechanism {}"
msgstr "不明なメカニズム {}"

#: dnstools/lib.py:179
msgid "Unknown modifier {}"
msgstr "不明な節 : {}"

#: dnstools/lib.py:182
msgid "Duplicate modifier {} found"
msgstr "モディフィアの重複 {}"

#: dnstools/lib.py:186
msgid "No mechanism found"
msgstr "メカニズムなし"

#: dnstools/lib.py:193
msgid "Not a valid DKIM record"
msgstr "有効な DKIM レコードなし"

#: dnstools/lib.py:201 dnstools/lib.py:254
msgid "Invalid tag {}"
msgstr "無効なタグ {}"

#: dnstools/lib.py:206
msgid "No key found in record"
msgstr "レコードにキーが見つかりません"

#: dnstools/lib.py:213
msgid "Wrong value {} for tag {}"
msgstr "タグ {} の値 {} に誤り"

#: dnstools/lib.py:228
msgid "Wrong value {} for tag {}:"
msgstr "タグ {} の値 {} に誤り : "

#: dnstools/lib.py:232
msgid " not an integer"
msgstr "整数ではありません"

#: dnstools/lib.py:234
msgid " less than {}"
msgstr "{} より小さい"

#: dnstools/lib.py:237
msgid " greater than {}"
msgstr "{} より大きい"

#: dnstools/lib.py:246
msgid "Not a valid DMARC record"
msgstr "有効な DMARC レコードではありません"

#: dnstools/lib.py:257
msgid "Unknown tag {}"
msgstr "不明なタグ {}"

#: dnstools/lib.py:262
msgid "Missing required p tag"
msgstr "必要な p タグが見つかりません"

#: dnstools/models.py:78
msgid "Public key mismatchs"
msgstr "パブリックキーが不一致です"

#: imap_migration/api/v2/serializers.py:117
msgid "Access denied"
msgstr "アクセス拒否"

#: imap_migration/app_settings.py:19
msgid "Enable IMAP Migration"
msgstr "IMAP 移行を有効に"

#: imap_migration/app_settings.py:20
msgid "Enable IMAP Migration."
msgstr "IMAP 移行を有効にします。"

#: imap_migration/app_settings.py:30
msgid "OfflineIMAP"
msgstr "オフライン IMAP"

#: imap_migration/app_settings.py:36
msgid "Concurrent sync jobs"
msgstr "同期ジョブの並行実行"

#: imap_migration/app_settings.py:38
msgid "The maximum number of concurrent synchronization jobs"
msgstr "同期ジョブの並行実行の最大値"

#: imap_migration/app_settings.py:49
msgid "OfflineIMAP Filter"
msgstr "オフライン IMAP フィルター"

#: imap_migration/app_settings.py:55
msgid "Create Folders"
msgstr "フォルダー作成"

#: imap_migration/app_settings.py:57
msgid "Allow Creation of missing folders during sync"
msgstr "同期処理の際に存在しないフォルダーを作成することを許可"

#: imap_migration/app_settings.py:64
msgid "Folder Filter Exclusions"
msgstr "除外フォルダーフィルター"

#: imap_migration/app_settings.py:66
msgid ""
"Use a regular expression to explicitly include folders in sync. Example: "
"^Trash$|Del"
msgstr "同期から除外するフォルダーを正規表現で明示します。例 : ^Trash$|Del"

#: imap_migration/app_settings.py:74
msgid "Folder Filter Inclusions"
msgstr "対象フォルダーフィルター"

#: imap_migration/app_settings.py:76
msgid ""
"A comma seperated list of folders to explicitly include in sync even if "
"filtered by the Folder Filter Exclusions. Example: debian.user, "
"debian.personal "
msgstr "除外フォルダーフィルターで対象から外れても同期に含めるフォルダーをカンマ区切りで明示します。例 : debian.user, debian.personal"

#: imap_migration/apps.py:20
msgid "IMAP Migration"
msgstr "IMAP 移行"

#: imap_migration/checks.py:12
msgid ""
"Automatic domain/mailbox creation is disabled which means IMAP "
"authentication won't work."
msgstr "IMAP 認証が動作しない場合にドメイン・メールボックスの自動作成を無効にします。"

#: imap_migration/checks.py:16
msgid ""
"Go to the online parameters panel (admin tab) and activate this feature."
msgstr "パラメーターパネル ( 管理者タブ ) でこの機能を有効にします。"

#: ldapsync/lib.py:82
msgid "Failed to create LDAP account: {}"
msgstr "LDAP アカウントの作成に失敗 : {}"

#: ldapsync/lib.py:121
msgid "Failed to update LDAP account: {}"
msgstr "LDAP アカウントの更新に失敗 : {}"

#: ldapsync/lib.py:135
msgid "Failed to delete LDAP account: {}"
msgstr "LDAP アカウントの削除に失敗 : {}"

#: ldapsync/lib.py:144
msgid "Failed to disable LDAP account: {}"
msgstr "LDAP アカウントの無効化に失敗 : {}"

#: lib/api_client.py:35 lib/api_client.py:67 lib/api_client.py:81
#: lib/api_client.py:85
#, python-format
msgid "Failed to communicate with public API: %s"
msgstr "パブリック API での通信に失敗 : %s"

#: lib/email_utils.py:140 lib/email_utils.py:351
msgid "unable to determine encoding of string"
msgstr "文字列のエンコード検出ができませんでした"

#: lib/exceptions.py:83
msgid "Permission denied: {}"
msgstr "アクセス拒否 : {}"

#: lib/exceptions.py:84
msgid "Permission denied"
msgstr "アクセス拒否"

#: lib/fields.py:13 lib/fields.py:26
msgid "Enter a valid email address."
msgstr "有効なメールアドレスを入力してください"

#: lib/ldap_utils.py:126
msgid "Failed to update password: {}"
msgstr "パスワードの更新に失敗しました:  {}"

#: lib/validators.py:13
msgid "Enter a valid domain name"
msgstr "有効なドメイン名を入力してください"

#: limits/app_settings.py:13
msgid "Default per-admin limits"
msgstr "管理者ごとの制限のデフォルト"

#: limits/app_settings.py:19
msgid "Enable per-admin limits"
msgstr "管理者ごとの制限を有効に"

#: limits/app_settings.py:20
msgid "Enable or disable per-admin limits"
msgstr "管理者ごとの制限を有効もしくは無効にします"

#: limits/app_settings.py:26 limits/app_settings.py:111 limits/constants.py:47
#: limits/constants.py:101
msgid "Domain admins"
msgstr "ドメイン管理者数"

#: limits/app_settings.py:29
msgid ""
"Maximum number of allowed domain administrators for a new administrator. (0 "
"to deny any creation, -1 to allow unlimited creations)"
msgstr "新しい管理者に作成を許可するドメイン管理者数の上限。( 0 で作成の拒否、-1 で無制限の作成 )"

#: limits/app_settings.py:41
msgid ""
"Maximum number of allowed domains for a new administrator. (0 to deny any "
"creation, -1 to allow unlimited creations)"
msgstr "新しい管理者に作成を許可するドメイン数の上限。( 0 で作成の拒否、-1 で無制限の作成 )"

#: limits/app_settings.py:49 limits/app_settings.py:123 limits/constants.py:22
#: limits/constants.py:76
msgid "Domain aliases"
msgstr "ドメインエイリアス"

#: limits/app_settings.py:52
msgid ""
"Maximum number of allowed domain aliases for a new administrator. (0 to deny"
" any creation, -1 to allow unlimited creations)"
msgstr "新しい管理者に作成を許可するドメインエイリアス数の上限。( 0 で作成の拒否、-1 で無制限の作成 )"

#: limits/app_settings.py:64
msgid ""
"Maximum number of allowed mailboxes for a new administrator. (0 to deny any "
"creation, -1 to allow unlimited creations)"
msgstr "新しい管理者に作成を許可するメールボックス数の上限。( 0 で作成の拒否、-1 で無制限の作成 )"

#: limits/app_settings.py:73 limits/app_settings.py:146 limits/constants.py:39
#: limits/constants.py:92
msgid "Mailbox aliases"
msgstr "メールボックス エイリアス"

#: limits/app_settings.py:76
msgid ""
"Maximum number of allowed aliases for a new administrator. (0 to deny any "
"creation, -1 to allow unlimited creations)"
msgstr "新しい管理者に作成を許可するエイリアス数の上限。( 0 で作成の拒否、-1 で無制限の作成 )"

#: limits/app_settings.py:84 limits/constants.py:59
msgid "Quota"
msgstr "容量制限"

#: limits/app_settings.py:86
msgid ""
"The quota a reseller will be allowed to share between the domains he "
"creates. (0 means no quota)"
msgstr "リセラーに対して許可する、作成するドメインの容量制限の合算値。( 0 で無制限 )"

#: limits/app_settings.py:98
msgid "Default per-domain limits"
msgstr "ドメインごとの容量制限のデフォルト"

#: limits/app_settings.py:104
msgid "Enable per-domain limits"
msgstr "ドメインごとの容量制限を有効に"

#: limits/app_settings.py:105
msgid "Enable or disable per-domain limits"
msgstr "ドメインごとの容量制限を有効もしくは無効にします"

#: limits/app_settings.py:114
msgid ""
"Maximum number of allowed domain administrators for a new domain. (0 to deny"
" any creation, -1 to allow unlimited creations)"
msgstr "新しいドメインに作成を許可するドメイン管理者数の上限。( 0 で作成の拒否、-1 で無制限の作成 )"

#: limits/app_settings.py:126
msgid ""
"Maximum number of allowed domain aliases for a new domain. (0 to deny any "
"creation, -1 to allow unlimited creations)"
msgstr "新しいドメインに作成を許可するドメインエイリアス数の上限。( 0 で作成の拒否、-1 で無制限の作成 )"

#: limits/app_settings.py:138
msgid ""
"Maximum number of allowed mailboxes for a new domain. (0 to deny any "
"creation, -1 to allow unlimited creations)"
msgstr "新しいドメインに作成を許可するメールボックス数の上限。( 0 で作成の拒否、-1 で無制限の作成 )"

#: limits/app_settings.py:149
msgid ""
"Maximum number of allowed aliases for a new domain. (0 to deny any creation,"
" -1 to allow unlimited creations)"
msgstr "新しいドメインに作成を許可するエイリアス数の上限。( 0 で作成の拒否、-1 で無制限の作成 )"

#: limits/apps.py:16
msgid "Limits"
msgstr "制限"

#: limits/constants.py:14
msgid "Maximum number of domains this user can create"
msgstr "このユーザーが作成できるドメイン数の上限"

#: limits/constants.py:23
msgid "Maximum number of domain aliases this user can create"
msgstr "このユーザーが作成できるドメインエイリアス数の上限"

#: limits/constants.py:32
msgid "Maximum number of mailboxes this user can create"
msgstr "このユーザーが作成できるメールボックス数の上限"

#: limits/constants.py:40
msgid "Maximum number of mailbox aliases this user can create"
msgstr "このユーザーが作成できるメールボックスエイリアス数の上限"

#: limits/constants.py:49
msgid "Maximum number of domain administrators this user can create"
msgstr "このユーザーが作成できるドメイン管理者数の上限"

#: limits/constants.py:60
msgid "Quota shared between domains of this reseller"
msgstr "このリセラーのドメイン全体の容量制限"

#: limits/constants.py:77
msgid "Maximum number of domain aliases allowed for this domain."
msgstr "このドメインに許可するドメインエイリアス数の上限"

#: limits/constants.py:85
msgid "Maximum number of mailboxes allowed for this domain."
msgstr "このドメインに許可するメールボックス数の上限"

#: limits/constants.py:93
msgid "Maximum number of mailbox aliases allowed for this domain."
msgstr "このドメインに許可するメールボックスエイリアス数の上限"

#: limits/constants.py:102
msgid "Maximum number of domain admins allowed for this domain."
msgstr "このドメインに許可するドメイン管理者数の上限"

#: limits/lib.py:19
#, python-format
msgid "%s: limit reached"
msgstr "%s: 制限に到達"

#: limits/lib.py:29
msgid "Not enough resources"
msgstr "リソース不足"

#: limits/lib.py:46 limits/models.py:46
msgid "You're not allowed to define unlimited values"
msgstr "無制限の値を設定する権限がありません"

#: limits/models.py:56 limits/models.py:119
msgid "unlimited"
msgstr "無制限"

#: maillog/api/v2/serializers.py:35
msgid "You must provide start and end dates when period is custom"
msgstr "期間をカスタマイズする際は、開始日と終了日を設定してください"

#: maillog/apps.py:20
msgid "Path to the log file"
msgstr "ログファイルのパス"

#: maillog/apps.py:22
msgid "Path to log file used to collect statistics"
msgstr "ログファイルのパス。統計を集計する際に利用します"

#: maillog/apps.py:29
msgid "Directory to store RRD files"
msgstr "RRD ファイルを保存ディレクトリー"

#: maillog/apps.py:31
msgid "Path to directory where RRD files are stored"
msgstr "RRD ファイルを保存するディレクトリーのパス"

#: maillog/apps.py:38
msgid "Show greylisted messages"
msgstr "greylist のメッセージを表示"

#: maillog/apps.py:40
msgid "Differentiate between hard and soft rejects (greylisting)"
msgstr "ハード拒否とソフト拒否を区別する ( greylist )"

#: maillog/apps.py:60
msgid "Statistics"
msgstr "統計"

#: maillog/graphics.py:91
msgid "Failed to locate rrdtool binary."
msgstr "rrdtool バイナリーの場所に到達できません。"

#: maillog/graphics.py:166
msgid "Average traffic (msgs/min)"
msgstr "平均通信量 ( メッセージ / 分 )"

#: maillog/graphics.py:169
msgid "sent messages"
msgstr "送信メッセージ"

#: maillog/graphics.py:170
msgid "received messages"
msgstr "受信メッセージ"

#: maillog/graphics.py:171
msgid "bounced messages"
msgstr "バウンスメッセージ"

#: maillog/graphics.py:172
msgid "rejected messages"
msgstr "拒否メッセージ"

#: maillog/graphics.py:173
msgid "virus messages"
msgstr "ウイルスメッセージ"

#: maillog/graphics.py:174
msgid "spam messages"
msgstr "SPAM メッセージ"

#: maillog/graphics.py:181
msgid "greylisted messages"
msgstr "greylist メッセージ"

#: maillog/graphics.py:198
msgid "Average normal traffic size (bytes/min)"
msgstr "通常トラフィックサイズの平均値 ( bytes / 分 )"

#: maillog/graphics.py:201
msgid "received size"
msgstr "受信サイズ"

#: maillog/graphics.py:202
msgid "sent size"
msgstr "送信サイズ"

#: maillog/graphics.py:209
msgid "Mail traffic"
msgstr "メールトラフィック"

#: maillog/graphics.py:252
msgid "Average account creation (account/hour)"
msgstr "アカウント作成数の平均 ( アカウント / 時間 )"

#: maillog/graphics.py:254
msgid "New accounts"
msgstr "新しいアカウント"

#: maillog/graphics.py:261
msgid "Accounts"
msgstr "アカウント"

#: pdfcredentials/api/v2/serializers.py:35 pdfcredentials/documents.py:60
msgid "Personal account information"
msgstr "個人アカウント情報"

#: pdfcredentials/api/v2/serializers.py:86
msgid "Directory is not writable"
msgstr "ディレクトリーが書き込み禁止"

#: pdfcredentials/api/v2/serializers.py:114
msgid "No document available for this user"
msgstr "このユーザーのための文書なし"

#: pdfcredentials/app_settings.py:19
msgid "Enable PDF credentials"
msgstr "認証情報 PDF を有効に"

#: pdfcredentials/app_settings.py:21
msgid "Enabled PDF credentials document generation on account creation."
msgstr "アカウント作成時に認証情報 PDF の文書を有効にします。"

#: pdfcredentials/app_settings.py:32
msgid "Documents storage"
msgstr "文書保存場所"

#: pdfcredentials/app_settings.py:38
msgid "Directory to save documents into"
msgstr "文書を保存するディレクトリー"

#: pdfcredentials/app_settings.py:40
msgid "Path to a directory where PDF documents will be saved"
msgstr "PDF 文書の保存先ディレクトリーパス"

#: pdfcredentials/app_settings.py:51
msgid "Security options"
msgstr "セキュリティー設定"

#: pdfcredentials/app_settings.py:57
msgid "Delete documents after the first download"
msgstr "ダウンロードしたら文書を削除"

#: pdfcredentials/app_settings.py:59
msgid ""
"Automatically delete a document just after its first download from this "
"interface"
msgstr "このインターフェースからダウンロードをしたのちに、文書を自動的に削除します"

#: pdfcredentials/app_settings.py:68
msgid "Generate documents only at account creation"
msgstr "アカウント作成時のみ文書を作成"

#: pdfcredentials/app_settings.py:71
msgid ""
"Generate a new document only when a new account is created. If set to no, a "
"new document will be created each time a password is updated."
msgstr "新しくアカウントを作成する際にのみ、新しい文書を作成します。「いいえ」に設定すると、パスワードを更新するたびに新しい文書を作成します。"

#: pdfcredentials/app_settings.py:84
msgid "Customization options"
msgstr "カスタマイズ設定"

#: pdfcredentials/app_settings.py:90
msgid "Title"
msgstr "タイトル"

#: pdfcredentials/app_settings.py:91
msgid "The document's title"
msgstr "文書のタイトル"

#: pdfcredentials/app_settings.py:97
msgid "Web panel url"
msgstr "Web パネル URL"

#: pdfcredentials/app_settings.py:98
msgid "URL of the Modoboa web panel"
msgstr "Modoboa Web パネルの URL"

#: pdfcredentials/app_settings.py:104
msgid "Custom message"
msgstr "カスタムメッセージ"

#: pdfcredentials/app_settings.py:106
msgid "A custom message that will appear at the end of documents"
msgstr "文書の最後に記載するカスタムメッセージ"

#: pdfcredentials/app_settings.py:113
msgid "Include mail client connection settings"
msgstr "メールクライアント接続設定を含める"

#: pdfcredentials/app_settings.py:115
msgid ""
"Include required SMTP and IMAP connection information to configure a mail "
"client, a tablet or a phone"
msgstr "メールクライアント、タブレットやスマートフォンに設定する際に必要な SMTP・IMAP 接続の情報を含める"

#: pdfcredentials/app_settings.py:123 pdfcredentials/documents.py:133
msgid "SMTP server address"
msgstr "SMTP サーバーアドレス"

#: pdfcredentials/app_settings.py:126
msgid "Address of the SMTP server (hostname or IP)"
msgstr "SMTP サーバーのアドレス ( ホスト名か IP アドレス )"

#: pdfcredentials/app_settings.py:133 pdfcredentials/documents.py:134
msgid "SMTP server port"
msgstr "SMTP サーバーポート"

#: pdfcredentials/app_settings.py:135
msgid "Listening port of the SMTP server"
msgstr "SMTP サーバーのポート"

#: pdfcredentials/app_settings.py:141 pdfcredentials/documents.py:135
msgid "SMTP connection security"
msgstr "SMTP 接続セキュリティー"

#: pdfcredentials/app_settings.py:143 pdfcredentials/app_settings.py:169
msgid "Connection security mechanism"
msgstr "接続セキュリティーメカニズム"

#: pdfcredentials/app_settings.py:149 pdfcredentials/documents.py:136
msgid "IMAP server address"
msgstr "IMAP サーバー アドレス"

#: pdfcredentials/app_settings.py:152
msgid "Address of the IMAP server (hostname or IP)"
msgstr "IMAP サーバーのアドレス ( ホスト名か IP アドレス )"

#: pdfcredentials/app_settings.py:159 pdfcredentials/documents.py:137
msgid "IMAP server port"
msgstr "IMAP サーバーポート"

#: pdfcredentials/app_settings.py:161
msgid "Listening port of the IMAP server"
msgstr "IMAP サーバーのポート"

#: pdfcredentials/app_settings.py:167 pdfcredentials/documents.py:138
msgid "IMAP connection security"
msgstr "IMAP 接続セキュリティー"

#: pdfcredentials/apps.py:21
msgid "PDF Credentials"
msgstr "認証情報 PDF"

#: pdfcredentials/documents.py:65
msgid "Powered by Modoboa - Mail hosting made simple"
msgstr "Powered by Modoboa - メールホスティングをシンプルに"

#: pdfcredentials/documents.py:84
#, python-format
msgid ""
"\n"
"Dear %s, this document contains the credentials you will need\n"
"to connect to Modoboa. Learn the content and destroy\n"
"the document as soon as possible.\n"
msgstr "\n%s 様、この文書には Modoboa メールサーバーへの接続に必要な認証情報を記しています。\n認証情報が第三者に漏れれば、メールの内容をすべて読み取られるほか、なりすましによるメール送信も可能になります。\n内容を理解し必要な設定を終えたら、厳重に管理する・廃棄するなど、対応をおねがいします。\n"

#: pdfcredentials/documents.py:95
msgid "Web panel:"
msgstr "Web パネル : "

#: pdfcredentials/documents.py:98
msgid "Username"
msgstr "ユーザ名"

#: pdfcredentials/documents.py:99
msgid "Password"
msgstr "パスワード"

#: pdfcredentials/documents.py:116
msgid ""
"\n"
"Here you can view your emails anytime online, create filters or manage\n"
"your contacts.\n"
msgstr "\nこの画面からあなたのメールをオンラインで確認できるほか、表示フィルターを\n作成したり、連絡先を管理することができます。\n"

#: pdfcredentials/documents.py:126
msgid "Please change your password!"
msgstr "パスワードを変更してください！"

#: pdfcredentials/documents.py:130
msgid "PC/Tablet/Smartphone configuration:"
msgstr "パソコン・タブレット・スマートフォン設定 : "

#: pdfcredentials/documents.py:153
msgid "Use those settings for your computer, tablet or phone."
msgstr "この設定はパソコン、またはスマートフォンやタブレットで利用します。"

#: pdfcredentials/handlers.py:35
msgid ""
"Failed to create PDF_credentials directory. Please check the permissions or "
"the path."
msgstr "PDF_credentials ディレクトリーの作成に失敗しました。権限またはパスを確認してください。"

#: pdfcredentials/handlers.py:64
msgid "Download PDF credentials"
msgstr "認証情報 PDF をダウンロード"

#: pdfcredentials/lib.py:29
msgid "Can't write to selected directory to store PDF documents"
msgstr "PDF 文書を保存しようとしましたが、指定のディレクトリーに書き込めません"

#: pdfcredentials/lib.py:35
#, python-format
msgid "Failed to create the directory that will contain PDF documents (%s)"
msgstr "PDF 文書を保存するディレクトリーの作成に失敗 ( %s )"

#: pdfcredentials/lib.py:82
msgid ""
"Failed to create PDF_credentials file. Please check the permissions or the "
"path."
msgstr "認証情報 PDF ファイルの作成に失敗。権限またはパスを確認してください。"

#: policyd/core.py:99
msgid "Daily sending limit reached"
msgstr "1 日の送信通数制限に到達"

#: policyd/core.py:118
msgid "domain"
msgstr "ドメイン"

#: policyd/core.py:136
msgid "[modoboa] Sending limit reached"
msgstr "[modoboa] 送信通数制限に到達"

#: policyd/templates/policyd/notifications/limit_reached.html:3
#, python-format
msgid "Daily limit of the following %(ltype)s has been reached: %(name)s."
msgstr "%(ltype)s にもとづく 1 日あたりの制限に到達しました : %(name)s"

#: relaydomains/api/v1/serializers.py:24
msgid "Unsupported service"
msgstr "サポート外サービス"

#: relaydomains/transport.py:16
msgid "target host address"
msgstr "対象ホストアドレス"

#: relaydomains/transport.py:21
msgid "target host port"
msgstr "対象ホストポート"

#: relaydomains/transport.py:27
msgid "verify recipients"
msgstr "受信者の確認"

#: rspamd/api/v2/serializers.py:15
msgid "Provided path is not writable."
msgstr "指定のパスは書き込みできません。"

#: rspamd/api/v2/serializers.py:18
msgid "Please create the parent directory for the files"
msgstr "該当のファイル用の上位ディレクトリーを作成してください"

#: rspamd/api/v2/serializers.py:22
msgid "Unable to write to the directory."
msgstr "そのディレクトリーには書き込みできません。"

#: rspamd/app_settings.py:13
msgid "DKIM signing settings"
msgstr "DKIM 署名設定"

#: rspamd/app_settings.py:19
msgid "Key map path"
msgstr "キー マップ パス"

#: rspamd/app_settings.py:21
msgid ""
"Absolute path of the file which contains paths to DKIM private keys. Must be"
" readable by _rspamd group or user."
msgstr "DKIM プライベートキー ファイルの絶対パス。_rspamd グループまたはユーザーに読み取り権限が必要です。"

#: rspamd/app_settings.py:30
msgid "Selector map path"
msgstr "セレクター マップ パス"

#: rspamd/app_settings.py:32
msgid ""
"Absolute path of the file which contains names of DKIM selectors. Must be "
"readable by _rspamd group or user."
msgstr "DKIM セレクター ファイルの絶対パス。_rspamd グループまたはユーザーに読み取り権限が必要です。"

#: rspamd/app_settings.py:51
msgid "Rspamd dashboard location"
msgstr "Rspamd ダッシュボードの場所"

#: rspamd/app_settings.py:53
msgid ""
"Location of the rspamd dashboard. Can either be a relative URL (eg. "
"'/rspamd') or a full URL (eg. 'https://rspamd.domain.tld/').Only use for "
"guidance on the Super-Admin dashboard."
msgstr "rspamd ダッシュボードの場所。相対 URL ( 例 : '/rspamd' ) またはフル URL ( 例 : 'https://rspamd.domain.tld/' ) のどちらでもかまいません。スーパー管理者ダッシュボードからのガイドにしか利用しません。"

#: rspamd/apps.py:18
msgid "Rspamd"
msgstr "Rspamd"

#: sievefilters/api/v2/viewsets.py:193
#, python-format
msgid "Filter %s already exists"
msgstr "フィルター、%s はすでに存在します"

#: sievefilters/app_settings.py:12
msgid "ManageSieve settings"
msgstr "ManageSieve 設定"

#: sievefilters/app_settings.py:19
msgid "Address of your MANAGESIEVE server"
msgstr "MANAGESIEVE サーバーのアドレス"

#: sievefilters/app_settings.py:27
msgid "Listening port of your MANAGESIEVE server"
msgstr "MANAGESIEVE サーバーのポート"

#: sievefilters/app_settings.py:34
msgid "Connect using STARTTLS"
msgstr "STARTTLS で接続"

#: sievefilters/app_settings.py:35
msgid "Use the STARTTLS extension"
msgstr "STARTTLS 拡張を使用"

#: sievefilters/app_settings.py:45 webmail/app_settings.py:30
msgid "IMAP settings"
msgstr "IMAP 設定"

#: sievefilters/app_settings.py:52 webmail/app_settings.py:38
msgid "Address of your IMAP server"
msgstr "IMAP サーバーのアドレス"

#: sievefilters/app_settings.py:60 webmail/app_settings.py:47
msgid "Use a secured connection to access IMAP server"
msgstr "IMAP サーバーに暗号化接続する"

#: sievefilters/app_settings.py:68 webmail/app_settings.py:56
msgid "Listening port of your IMAP server"
msgstr "IMAP サーバーのポート"

#: sievefilters/apps.py:15
msgid "Sieve filters"
msgstr "Sieve フィルター"

#: sievefilters/constants.py:10
msgid "All of the following"
msgstr "以下のすべて"

#: sievefilters/constants.py:11
msgid "Any of the following"
msgstr "以下のいずれか"

#: sievefilters/constants.py:12
msgid "All messages"
msgstr "すべてのメッセージ"

#: sievefilters/constants.py:16
msgid "contains"
msgstr "含む"

#: sievefilters/constants.py:17
msgid "does not contain"
msgstr "含まない"

#: sievefilters/constants.py:18
msgid "is"
msgstr "一致する"

#: sievefilters/constants.py:19
msgid "is not"
msgstr "一致しない"

#: sievefilters/constants.py:23 webmail/lib/imapemail.py:304
msgid "Subject"
msgstr "件名"

#: sievefilters/constants.py:24
msgid "Sender"
msgstr "送信者"

#: sievefilters/constants.py:26
msgid "Cc"
msgstr "CC"

#: sievefilters/constants.py:29
msgid "Size"
msgstr "サイズ"

#: sievefilters/constants.py:31
msgid "is greater than"
msgstr " > "

#: sievefilters/constants.py:32
msgid "is less than"
msgstr " < "

#: sievefilters/constants.py:40
msgid "Move message to"
msgstr "メッセージの移動先"

#: sievefilters/constants.py:46 sievefilters/constants.py:60
msgid "Keep local copy"
msgstr "ローカルコピーを保持する"

#: sievefilters/constants.py:54
msgid "Redirect message to"
msgstr "メッセージのリダイレクト先"

#: sievefilters/constants.py:68
msgid "Reject message"
msgstr "メッセージを拒否"

#: sievefilters/constants.py:71
msgid "Stop processing"
msgstr "処理を停止"

#: sievefilters/lib.py:49
msgid "Connection to MANAGESIEVE server failed, check your configuration"
msgstr "MANAGESIEVE サーバーへの接続に失敗。設定を確認してください。"

#: sievefilters/lib.py:76
msgid "Not enough space on server"
msgstr "サーバーに十分な空きがありません"

#: templates/404.html:4 templates/500.html:4
msgid "Sorry"
msgstr "すみません"

#: templates/404.html:5
msgid "The requested page does not exist."
msgstr "リクエストされたページはありません。"

#: templates/500.html:5
msgid "An internal error occured."
msgstr "内部エラーが発生しました"

#: templates/common/error.html:4
msgid "Error:"
msgstr "エラー："

#: templates/common/generic_field.html:11
#: templates/common/generic_fields_group.html:8
msgid "Help:"
msgstr "ヘルプ："

#: templates/registration/base.html:8
msgid "Welcome to Modoboa"
msgstr "Modoboa へようこそ"

#: templates/registration/login.html:11
msgid "Identification"
msgstr "認証"

#: templates/registration/login.html:24
msgid "Remember me"
msgstr "記憶する"

#: templates/registration/login.html:26
msgid "Log in"
msgstr "ログイン"

#: templates/registration/login.html:29
#: templates/registration/password_reset_confirm_code.html:7
#: templates/registration/password_reset_done.html:7
#: templates/registration/password_reset_form.html:17
msgid "Forgot password?"
msgstr "パスワードがわからない？"

#: templates/registration/password_reset_complete.html:8
#, python-format
msgid ""
"Your password has been set. You may go ahead and <a href=\"%(url)s\">sign "
"in</a> now."
msgstr "パスワードを設定しました。<a href=\"%(url)s\">サインイン</a> で進みます。"

#: templates/registration/password_reset_confirm.html:8
msgid "Change password"
msgstr "パスワードを変更"

#: templates/registration/password_reset_confirm.html:25
#: templates/registration/password_reset_confirm_code.html:27
#: templates/registration/password_reset_form.html:23
msgid "Submit"
msgstr "送信"

#: templates/registration/password_reset_confirm.html:29
msgid ""
"The password reset link was invalid, possibly because it has already been "
"used. Please request a new password reset."
msgstr "パスワードの再設定リンクが無効です。おそらくこのリンクからはすでに操作を終えたものと思われます。もう一度「パスワード再設定」の操作をしてください。"

#: templates/registration/password_reset_confirm_code.html:22
msgid "Enter the code you've just received by SMS"
msgstr "SMS で受信したコードを入力してください"

#: templates/registration/password_reset_confirm_code.html:30
msgid "Resend code"
msgstr "コードの再送信"

#: templates/registration/password_reset_confirm_code.html:45
msgid "The code has been sent again"
msgstr "コードを再送信しました"

#: templates/registration/password_reset_done.html:8
msgid ""
"<p>We've emailed you instructions for setting your password, if an account "
"exists with the email you entered. You should receive them shortly.</p> "
"<p>If you don't receive an email, please make sure you've entered your "
"primary address, and check your spam folder.</p>"
msgstr "<p>入力したメールアドレスが登録済みのものであれば、パスワード設定方法を記載したメールが届きます。</p> <p>メールが届かなければ、入力したアドレスが正しいものだったかどうか、または SPAM フォルダーに配信されていないかを確認してください。</p>"

#: templates/registration/password_reset_email.html:4
#: templates/registration/password_reset_email_v2.html:4
#, python-format
msgid ""
"To initiate the password reset process for your %(username)s Modoboa "
"account, click the link below:"
msgstr "パスワードの再設定処理をはじめます。対象のアカウントは %(username)s でよろしいですか？　よろしければ以下のリンクをクリックしてください : "

#: templates/registration/password_reset_email.html:10
#: templates/registration/password_reset_email_v2.html:10
msgid ""
"If clicking the link above doesn't work, please copy and paste the URL in a "
"new browser window instead."
msgstr "上のリンクが動作しなければ、URL を Web ブラウザにコピー・貼りつけしてください。"

#: templates/registration/password_reset_email.html:14
#: templates/registration/password_reset_email_v2.html:14
msgid "Sincerely, The Modoboa Team."
msgstr "Modoboa チームより"

#: templates/registration/password_reset_subject.txt:3
msgid "Modoboa password reset"
msgstr "Modoboa メールサーバー パスワード再設定"

#: templates/registration/twofactor_code_verify.html:11
msgid "Verification"
msgstr "確認"

#: templates/registration/twofactor_code_verify.html:15
msgid "Use TOTP/backup code"
msgstr "2FA コード・再設定コードを使用"

#: templates/registration/twofactor_code_verify.html:18
msgid "Use Webauthn device"
msgstr "パスキー デバイスを使用"

#: templates/registration/twofactor_code_verify.html:27
msgid "One-time password"
msgstr "ワンタイム パスワード"

#: templates/registration/twofactor_code_verify.html:35
msgid ""
"Enter the code from the two-factor app on your mobile device. If you've lost"
" your device, you may enter one of your recovery codes."
msgstr "あなたのスマートフォンの 2 要素認証アプリのコードを入力してください。スマートフォンをなくしているなどの場合は、再設定コードのいずれかを入力してください。"

#: templates/registration/twofactor_code_verify.html:42
msgid "Verify code"
msgstr "確認コード"

#: templates/registration/twofactor_code_verify.html:45
#: templates/registration/twofactor_code_verify.html:51
msgid "Back"
msgstr "戻る"

#: templates/registration/twofactor_code_verify.html:49
msgid "Log using Webauthn"
msgstr "パスキーでログイン"

#: transport/backends.py:55
msgid "Invalid value"
msgstr "無効な値"

#: transport/models.py:15
msgid "pattern"
msgstr "パターン"

#: transport/models.py:16
msgid "service"
msgstr "サービス"

#: transport/models.py:17
msgid "next hop"
msgstr "次のホップ"

#: webmail/app_settings.py:17
msgid "Maximum attachment size"
msgstr "最大添付サイズ"

#: webmail/app_settings.py:19
msgid "Maximum attachment size in bytes (or KB, MB, GB if specified)"
msgstr "添付ファイルの最大サイズをバイトで ( 必要に応じて KB, MB, GB も指定可 ) "

#: webmail/app_settings.py:67
msgid "SMTP settings"
msgstr "SMTP 設定"

#: webmail/app_settings.py:75
msgid "Address of your SMTP server"
msgstr "SMTP サーバーのアドレス"

#: webmail/app_settings.py:82
msgid "Secured connection mode"
msgstr "暗号化接続モード"

#: webmail/app_settings.py:84
msgid "Use a secured connection to access SMTP server"
msgstr "SMTP サーバーに暗号化接続する"

#: webmail/app_settings.py:93
msgid "Listening port of your SMTP server"
msgstr "SMTP サーバーのポート"

#: webmail/app_settings.py:100
msgid "Authentication required"
msgstr "認証が必要"

#: webmail/app_settings.py:102
msgid "Server needs authentication"
msgstr "サーバーが認証を求めています"

#: webmail/app_settings.py:118
msgid "Display"
msgstr "表示"

#: webmail/app_settings.py:124
msgid "Default message display mode"
msgstr "デフォルトのメッセージ表示モード"

#: webmail/app_settings.py:126
msgid "The default mode used when displaying a message"
msgstr "メッセージを表示するモードのデフォルト値"

#: webmail/app_settings.py:133
msgid "Enable HTML links display"
msgstr "HTML リンク表示を有効に"

#: webmail/app_settings.py:136
msgid "Enable/Disable HTML links display"
msgstr "HTML リンクの表示の有効・無効を設定します"

#: webmail/app_settings.py:144
msgid "Number of displayed emails per page"
msgstr "1 ページに表示するメールの数"

#: webmail/app_settings.py:147
msgid "Sets the maximum number of messages displayed in a page"
msgstr "1 ページに表示するメッセージの最大数を設定します"

#: webmail/app_settings.py:154
msgid "Listing refresh rate"
msgstr "一覧の表示更新間隔"

#: webmail/app_settings.py:156
msgid "Automatic listing refresh rate (in secconds)"
msgstr "自動で一覧の表示を更新する間隔 ( 秒 )"

#: webmail/app_settings.py:163
msgid "Folder container's width"
msgstr "フォルダーコンテナーの幅"

#: webmail/app_settings.py:165
msgid "The width of the folder list container"
msgstr "フォルダーリストコンテナの幅"

#: webmail/app_settings.py:176
msgid "Folders"
msgstr "フォルダー"

#: webmail/app_settings.py:182
msgid "Trash folder"
msgstr "ごみ箱"

#: webmail/app_settings.py:184
msgid "Folder where deleted messages go"
msgstr "削除メッセージの行き先フォルダー"

#: webmail/app_settings.py:191
msgid "Sent folder"
msgstr "送信済みフォルダー"

#: webmail/app_settings.py:193
msgid "Folder where copies of sent messages go"
msgstr "送信済みメッセージの行き先フォルダー"

#: webmail/app_settings.py:200
msgid "Drafts folder"
msgstr "下書きフォルダー"

#: webmail/app_settings.py:201
msgid "Folder where drafts go"
msgstr "下書きの行き先フォルダー"

#: webmail/app_settings.py:207
msgid "Junk folder"
msgstr "SPAM フォルダー"

#: webmail/app_settings.py:209
msgid "Folder where junk messages should go"
msgstr "SPAM メッセージの行き先フォルダー"

#: webmail/app_settings.py:220
msgid "Composing messages"
msgstr "メッセージの作成"

#: webmail/app_settings.py:226
msgid "Default editor"
msgstr "デフォルトエディター"

#: webmail/app_settings.py:228
msgid "The default editor to use when composing a message"
msgstr "メッセージを作成するエディターのデフォルト値"

#: webmail/app_settings.py:235
msgid "Signature text"
msgstr "署名テキスト"

#: webmail/app_settings.py:237
msgid "User defined email signature"
msgstr "ユーザー定義のメール署名"

#: webmail/exceptions.py:18
msgid "Server response"
msgstr "サーバー応答"

#: webmail/lib/attachments.py:112
msgid "Failed to remove attachment: "
msgstr "添付ファイルの削除に失敗 : "

#: webmail/lib/imapemail.py:272
msgid "wrote:"
msgstr " : "

#: webmail/lib/imapemail.py:306
msgid "Date"
msgstr "日付"

#: webmail/lib/imapemail.py:322 webmail/lib/imapemail.py:325
msgid "Original message"
msgstr "元メッセージ"

#: webmail/lib/imaputils.py:221
msgid "Failed to retrieve hierarchy delimiter"
msgstr "階層区切りの取得に失敗"

#: webmail/lib/imaputils.py:472
msgid "Inbox"
msgstr "受信トレイ"

#: webmail/lib/imaputils.py:476
msgid "Drafts"
msgstr "下書き"

#: webmail/lib/imaputils.py:481
msgid "Junk"
msgstr "SPAM"

#: webmail/lib/imaputils.py:486
msgid "Sent"
msgstr "送信済み"

#: webmail/lib/imaputils.py:491
msgid "Trash"
msgstr "ごみ箱"

#: webmail/viewsets.py:343
#, python-format
msgid "Attachment is too big (limit: %s)"
msgstr "添付ファイルが大きすぎます ( 最大 : %s )"
